components/konflux-ui/staging/stone-stg-rh01/kustomization.yaml

```diff
@@ -13,6 +13,7 @@ configMapGenerator:
   literals:
     - IMPERSONATE=true
     - TEKTON_RESULTS_URL=https://tekton-results-api-service.tekton-results.svc.cluster.local:8080
+    - KUBEARCHIVE_URL=https://kubearchive-api-server.product-kubearchive.svc.cluster.local:8080
```

Comment:
can you please update the internal staging cluster as well? since the changes you made to the nginx config will take place in both staging clusters.

Reply:
included